.circleci/config.yml

```diff
@@ -242,10 +288,16 @@ jobs:
       - restore_cache: *restore_integration_cache
       - run: *check_cache
       - run: *install_cached_packages
+      - run: *test_junit
       - run: *build_autoinst
       - run:
           command: |
-            eval "$(t/test_postgresql | grep TEST_PG=)" && PERL5LIB=lib SCHEDULER_FULLSTACK=1 OPENQA_BASEDIR=. prove -v t/05-scheduler-full.t
+            eval "$(t/test_postgresql | grep TEST_PG=)" && PERL5LIB=lib SCHEDULER_FULLSTACK=1 OPENQA_BASEDIR=. prove --harness TAP::Harness::JUnit t/05-scheduler-full.t
+      - store_test_results:
+          path: test-results/junit
+      - store_artifacts:
+          path: test-results/t
+          destination: artifacts
```

Comment:
as this part is duplicated so often, can you please extract it and reuse it?

Reply:
Well, I was kinda assuming that this would be taken care of in the refactor. With the current configuration I can't avoid specifying the steps explicitly... although I could save  a couple lines I guess